spec/models/occupation_standard_spec.rb

```diff
@@ -456,4 +456,19 @@
       expect(occupation_standard.work_processes_hours_in_human_format).to eq "160"
     end
   end
+
+  describe "#public_document?" do
+    it "returns true if OccupationStandard#public_document flag is true" do
+      occupation_standard = build(:occupation_standard, public_document: true)
+
+      expect(occupation_standard.public_document?).to be true
+    end
+
+    it "trus true if associated standard import is public document regardless of public_document flag" do
+      occupation_standard = build(:occupation_standard, public_document: false)
+      allow_any_instance_of(OccupationStandard).to receive(:public_document?).and_return(true)
+
+      expect(occupation_standard.public_document?).to be true
```

Comment:
It seems we are asserting on the same method we're stubbing above, which means, if we break the implementation, this test will not fail. Did you mean to stub the return of standards_import?


```suggestion
      allow_any_instance_of(StandardImport).to receive(:public_document?).and_return(true)

      expect(occupation_standard.public_document?).to be true
```

Reply:
Good catch! You are correct. I meant to stub StandardImport